test/lit/passes/outlining.wast

```diff
@@ -1164,3 +1164,35 @@
     )
   )
 )
+
+;; Tests TryTable instructions are correctly filtered from being outlined.
+;; The (drop (i32.const 0)) instructions were added to form an outlineable
+;; sequence with the block that contains the try_table.
+(module
```

Comment:
Looks like we need to run the test update script to get the output here.

Reply:
oops, fixed, thanks